package-lock.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "EZID_UI_library",
+  "name": "ezid-ui-toolkit",
```

Comment:
Why we need this change?

Reply:
This change is from `npm install` and it matches the information in the file `package.json`.  I do not know why this was called "EZID_UI_library" before.  I did not change this, but `npm install` (or `yarn install`) must've changed it.

The contents of the file `package-lock.json` is automatically managed by npm.  It can regenerate or change when `npm install` is run and indicates the versions of packages that are installed.  IDK, why this was `EZID_UI_library` before since there is no package that matches that in `package.json` (even the main branch), so I'm not sure what it was referencing?